<HTML>
<HEAD>
<meta charset="UTF-8">
<title>UnknownCalendar - corda</title>
<link rel="stylesheet" href="../../../style.css">
</HEAD>
<BODY>
<a href="../../index.html">corda</a>&nbsp;/&nbsp;<a href="../index.html">net.corda.finance.workflows.utils</a>&nbsp;/&nbsp;<a href="./index.html">UnknownCalendar</a><br/>
<br/>
<h1>UnknownCalendar</h1>
<code><span class="keyword">class </span><span class="identifier">UnknownCalendar</span>&nbsp;<span class="symbol">:</span>&nbsp;<a href="../../net.corda.core.flows/-flow-exception/index.html"><span class="identifier">FlowException</span></a></code>
<h3>Constructors</h3>
<table>
<tbody>
<tr>
<td>
<p><a href="-init-.html">&lt;init&gt;</a></p>
</td>
<td>
<code><span class="identifier">UnknownCalendar</span><span class="symbol">(</span><span class="identifier" id="net.corda.finance.workflows.utils.UnknownCalendar$<init>(kotlin.String)/name">name</span><span class="symbol">:</span>&nbsp;<a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html"><span class="identifier">String</span></a><span class="symbol">)</span></code></td>
</tr>
</tbody>
</table>
<h3>Inherited Properties</h3>
<table>
<tbody>
<tr>
<td>
<p><a href="../../net.corda.core.flows/-flow-exception/original-error-id.html">originalErrorId</a></p>
</td>
<td>
<code><span class="keyword">var </span><span class="identifier">originalErrorId</span><span class="symbol">: </span><a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-long/index.html"><span class="identifier">Long</span></a><span class="symbol">?</span></code>
<p>the ID backing <a href="../../net.corda.core.flows/-flow-exception/get-error-id.html">getErrorId</a>. If null it will be set dynamically by the flow framework when
    the exception is handled. This ID is propagated to counterparty flows, even when the <a href="../../net.corda.core.flows/-flow-exception/index.html">FlowException</a> is
    downgraded to an <a href="../../net.corda.core.flows/-unexpected-flow-end-exception/index.html">UnexpectedFlowEndException</a>. This is so the error conditions may be correlated later on.</p>
</td>
</tr>
</tbody>
</table>
<h3>Inherited Functions</h3>
<table>
<tbody>
<tr>
<td>
<p><a href="../../net.corda.core.flows/-flow-exception/get-error-id.html">getErrorId</a></p>
</td>
<td>
<code><span class="keyword">open</span> <span class="keyword">fun </span><span class="identifier">getErrorId</span><span class="symbol">(</span><span class="symbol">)</span><span class="symbol">: </span><a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-long/index.html"><span class="identifier">Long</span></a><span class="symbol">?</span></code></td>
</tr>
</tbody>
</table>
<h3>Extension Functions</h3>
<table>
<tbody>
<tr>
<td>
<p><a href="../../net.corda.core.utilities/kotlin.-any/context-logger.html">contextLogger</a></p>
</td>
<td>
<code><span class="keyword">fun </span><a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-any/index.html"><span class="identifier">Any</span></a><span class="symbol">.</span><span class="identifier">contextLogger</span><span class="symbol">(</span><span class="symbol">)</span><span class="symbol">: </span><span class="identifier">&lt;ERROR CLASS&gt;</span></code>
<p>When called from a companion object, returns the logger for the enclosing class.</p>
</td>
</tr>
</tbody>
</table>
</BODY>
</HTML>
